internal/cli/cmd/install.go

```diff
@@ -80,6 +80,7 @@ cilium install --context kind-cluster1 --cluster-id 1 --cluster-name cluster1
 	cmd.Flags().StringVar(&params.Azure.TenantID, "azure-tenant-id", "", "Tenant ID of Azure Service Principal to use for installing Cilium (will create one if none provided)")
 	cmd.Flags().StringVar(&params.Azure.ClientID, "azure-client-id", "", "Client (application) ID of Azure Service Principal to use for installing Cilium (will create one if none provided)")
 	cmd.Flags().StringVar(&params.Azure.ClientSecret, "azure-client-secret", "", "Client secret of Azure Service Principal to use for installing Cilium (will create one if none provided)")
+	cmd.Flags().BoolVar(&params.SkipRollback, "skip-rollback", false, "Skip install rollback on failure")
```

Comment:
Nit: would it make sense to have it as `--rollback=false` instead (and a default of `true`), to be consistent with `--wait`?

Reply:
makes sense, I'll flip it!